src/Hosting/Hosting/src/StaticWebAssets/StaticWebAssetsLoader.cs

```diff
@@ -48,7 +48,9 @@ internal static void UseStaticWebAssetsCore(IWebHostEnvironment environment, Str
     {
         try
         {
+#pragma warning disable IL2026 // RequiresUnreferencedCodeAttribute: string should not be trimmed away and thus is safe to use
```

Comment:
You'll want to use an `UnconditionalSuppressMessage` attribute instead. The reason being is that `#pragma warning disable` only works for the Roslyn analyzer. When the app is trimmed, the ILLinker won't be able to see the pragmas, and it will still warn.